commit 99cb2b5a1879
Author: Francois Marier <francois@mozilla.com>
Date:   Mon Aug 28 17:06:00 2017 -0700

    Bug 1394601 - Log an error when the Google API key is missing during Safe Browsing initialization. r=gcp
    
    MozReview-Commit-ID: DNwa9lR0J6K
    
    --HG--
    extra : rebase_source : 0a547c0cf88e1ed09268cbb810abaaca09e98531
---
 toolkit/components/url-classifier/SafeBrowsing.jsm | 1 +
 1 file changed, 1 insertion(+)

diff --git toolkit/components/url-classifier/SafeBrowsing.jsm toolkit/components/url-classifier/SafeBrowsing.jsm
index 051432393539..32cab18c187f 100644
--- toolkit/components/url-classifier/SafeBrowsing.jsm
+++ toolkit/components/url-classifier/SafeBrowsing.jsm
@@ -343,6 +343,7 @@ this.SafeBrowsing = {
       let googleKey = Services.urlFormatter.formatURL("%GOOGLE_API_KEY%").trim();
       if ((provider == "google" || provider == "google4") &&
           (!googleKey || googleKey == "no-google-api-key")) {
+        log("Missing Google API key, clearing updateURL and gethashURL.");
         updateURL= "";
         gethashURL= "";
       }
